<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProviderSettings</class>
 <widget class="QWidget" name="ProviderSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Online Storage</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Provider:</string>
        </property>
        <property name="buddy">
         <cstring>providerSelection</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="providerSelection">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="buddy">
         <cstring>providerUserName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="providerUserName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="buddy">
         <cstring>providerPwd</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="providerPwd"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Collection:</string>
        </property>
        <property name="buddy">
         <cstring>listCollection</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="KComboBox" name="listCollection">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fetchCollection">
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addCollection">
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeCollection">
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Url:</string>
        </property>
        <property name="buddy">
         <cstring>providerUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KLineEdit" name="providerUrl">
        <property name="toolTip">
         <string>The url to query. For zotero enter either &quot;users&quot; or &quot;groups&quot;</string>
        </property>
        <property name="text">
         <string>users</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>FileType:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KComboBox" name="fileTypeSelection"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sync Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Sync mode:</string>
          </property>
          <property name="buddy">
           <cstring>syncMode</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KComboBox" name="syncMode">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>Download only</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Upload only</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Synchronize Database</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="importAttachments">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Import Attachments</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="exportAttachments">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Export Attachments</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="addContactsToAkonadi">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Add Contacts to Akonadi</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="KComboBox" name="contactCollection">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QCheckBox" name="addEventsToAkonadi">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Add Events to Akonadi</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="KComboBox" name="eventCollection">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Merge mode:</string>
          </property>
          <property name="buddy">
           <cstring>mergeMode</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KComboBox" name="mergeMode">
          <item>
           <property name="text">
            <string>Merge manually</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>User server version</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Keep local changes</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QCheckBox" name="askDeletion">
          <property name="text">
           <string>Ask before local entries are deleted</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>providerSelection</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ProviderSettings</receiver>
   <slot>switchProvider(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>switchProvider(int)</slot>
 </slots>
</ui>
